source/_integrations/arcam_fmj.markdown

```diff
@@ -93,6 +35,12 @@ Use an IR blaster to send a command to turn the device on using these discrete c
  - Zone 1: Protocol: NEC1 Device: 16 Function: 123
  - Zone 2: Protocol: NEC1 Device: 23 Function: 123
 
+You can generate the raw, broadlink or other ir format string using [irgen](https://github.com/elupus/irgen) tool like: `irgen -i nec1 -d 16 0 123 -o broadlink_base64`
+
+To trigger this IR command add an automation on the event `arcam.turn_on` filtering on
+the entity_id of the media_player zone entity. This can be added using device automations
+or manually using normal automations.
```

Comment:
```suggestion
To trigger this IR command add an automation on the event `arcam.turn_on` filtering on
the `entity_id` of the `media_player` zone entity. This can be added using device automations
or manually using normal automations.
```